src/data_store/data_store_conduit.cpp

```diff
@@ -597,7 +619,7 @@ void data_store_conduit::build_preloaded_owner_map(const std::vector<int>& per_r
 
 #if 0
```

Comment:
Dead code should be deleted. If it serves a useful testing purpose, write a unit test. If it's meant for debugging/informational purposes, make it toggleable, since it will probably be more widely useful. Otherwise it can always be retrieved from the Git history if needed.